test/functional/test_framework/test_framework.py

```
@@ -1136,22 +1136,80 @@ def is_bdb_compiled(self):
+            self.pubKeyOperator = bls_ret['public']
+            self.keyOperator = bls_ret['secret']
+
+    def get_collateral_value(self) -> int:
+        return EVONODE_COLLATERAL if self.evo else MASTERNODE_COLLATERAL
+
+    def get_collateral_vout(self, node: TestNode, txid: str, should_assert: bool = True) -> int:
+        for txout in node.getrawtransaction(txid, 1)['vout']:
+            if txout['value'] == self.get_collateral_value():
+                return txout['n']
+        if should_assert:
+            assert False, f"Unable to find collateral vout from txid {txid}"
```

Ruff (0.11.9)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)